### DISCUSSION TITLE
실습코드 6-11

### DISCUSSION THREAD

**Discussion by ssoree912** [fsoftwareengineer/todo-application#115] [질문과 답변] [answered]
안녕하세요 작가님 eb create를 이용해서 환경설정하는 과정에서 문제가 생겨 질문 드립니다.
실습 코드를 보고 작성완료 후 Successfully launched environment: PROD-TODO-BACKEND2 라고 떴는데도 일라스틱 빈스톡 환경으로 가보면 아무것도 생성이 안되있습니다.
 Demo/ 에서 eb init , eb create 다 작성했는데도 생성이 안됩니다. 다른 분들 discussion 참조해서 재빌드도 해보고 eb create 도 재차해보았습니다. 에러 코드는 안나오는데 어디서 잘못됐는지 모르겠어서 접근을 어떻게 해야할지 모르겠습니다.

<img width="1512" alt="오류" src="https://user-images.githubusercontent.com/66052043/167408671-12eb0898-b69c-4f80-99e0-67ce5e57d2f8.png">

<img width="939" alt="3" src="https://user-images.githubusercontent.com/66052043/167409996-f7c8090d-5a72-4a96-8378-3e6b9915dde6.png">

![오루2](https://user-images.githubusercontent.com/66052043/167411446-5921b266-220f-4ff8-aed4-c8aed5dcb639.png)

**Comment by fsoftwareengineer** [comment, marked as answer]
Aws콘솔의 리젼이 터미널에 프린트된 리젼(us-west-2)와 동일한지 확인 해 주시기 바랍니다.

**Reply by ssoree912** [reply]
답변 감사합니다!
위와같은 문제는 해결했으나 (포스트맨에도 정상적으로 응답이 리턴이 됩니다) 플랫폼 환경이 deprecated라고 뜨면서 낙후된/만료된 플랫폼을 사용하고 있다고 뜨는건 플랫폼 환경을 java로 설정했기 때문인걸까요?
profile 설정도 다 해봤지만 별 다른 변화가 없었습니다.
